awswrangler/opensearch/_read.py

```diff
@@ -102,6 +103,9 @@ def search(
 
 
     """
+    if is_scroll and _is_serverless(client):
+        raise exceptions.NotFound("Scrolled search is not currently available for OpenSearch Serverless.")
```

Comment:
Should we rename this exception to something like `OperationUnsupported` or something? `NotFound` just sounds too vague honestly.

Reply:
Yeah I agree, I'll put something more descriptive . Atm those are just named after HTTP statuses returned from OpenSeach API (404 NotFound, 409 Conflict...)